File: solutions/1/main.go
package twosum

// twoSum return indices of the two numbers such that they add up to target.
func twoSum(nums []int, target int) []int {
	for i, v := range nums {
		for a, av := range nums {
			if i == a {
				continue
			}
			if v+av == target {
				return []int{i, a}
			}
		}
	}
	return []int{}
}

File: solutions/1/main.py
"1. Two Sum"

class Solution:
    "Contains functions needed to compute the result."

    def twoSum(self, nums, target):
        "Returns indices of the two numbers such that they add up to target."
        history = {}
        for index in range(len(nums)):
            check = target - int(nums[index])
            if check in history:
                return [history[check], index]
            history[int(nums[index])] = index
